src/services/importService.ts

```diff
@@ -47,12 +47,23 @@ class ImportService {
             collectionName = collections[0].name;
         }
         const filePaths = files.map((file) => (file.file as ElectronFile).path);
-        this.ElectronAPIs.setToUploadFiles(filePaths, collectionName);
+        this.ElectronAPIs.setToUploadFiles(filePaths);
+        this.ElectronAPIs.setToUploadCollection(collectionName);
     }
 
     updatePendingUploads(files: FileWithCollection[]) {
-        const filePaths = files.map((file) => (file.file as ElectronFile).path);
-        this.ElectronAPIs.updatePendingUploadsFilePaths(filePaths);
+        const filePaths = [];
+        for (const file of files) {
+            if (file.isLivePhoto) {
```

Comment:
let's call this `fileWithCollection`only , not to get confused with the actual file